libs/domain/search/src/services/index.ts

```diff
@@ -1,10 +1,11 @@
 // organize-imports-ignore
```

Comment:
Do we need it? And is it even working?

Reply:
IDK, but I remember that we had an issue in production without it, we should retest it